tmva/tmva/inc/TMVA/DNN/Architectures/Cuda/CudaMatrix.h

```diff
@@ -22,12 +22,6 @@
 // in case we compile C++ code with std-17 and cuda with lower standard
 // use experimental string_view, otherwise keep as is
 #include "RConfigure.h"
-#ifdef R__HAS_STD_STRING_VIEW
-#ifndef R__CUDA_HAS_STD_STRING_VIEW
-#undef R__HAS_STD_STRING_VIEW
-#define R__HAS_STD_EXPERIMENTAL_STRING_VIEW
```

Comment:
Isn't anything using these preprocessor defines?

Reply:
My gut feeling here is "pretty sure", but maybe @lmoneta can also comment

Reply:
I think these are not needed anymore. There were there to fix  conflict happening in past  Cuda version when including string_view backport

Reply:
Do we have any CUDA build in the CI?

Reply:
@lmoneta ?

Reply:
No, we don't have it!